xml/System.Text.RegularExpressions/CaptureCollection.xml

```diff
@@ -560,7 +575,8 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Removes all items from the collection.</summary>
```

Comment:
Since the collection is RO, this summary should also say the method will throw in all cases.